system/Debug/Toolbar/Collectors/Database.php

```diff
@@ -139,17 +139,20 @@ public function display(): array
 
             foreach ($query['trace'] as &$traceLine) {
                 // Clean up the file paths
-                $traceLine['file'] = str_ireplace(APPPATH, 'APPPATH/', $traceLine['file']);
-                $traceLine['file'] = str_ireplace(SYSTEMPATH, 'SYSTEMPATH/', $traceLine['file']);
-                if (defined('VENDORPATH')) {
-                    // VENDORPATH is not defined unless `vendor/autoload.php` exists
-                    $traceLine['file'] = str_ireplace(VENDORPATH, 'VENDORPATH/', $traceLine['file']);
+                if (isset($traceLine['file'])) {
+                    $traceLine['file'] = str_ireplace(APPPATH, 'APPPATH/', $traceLine['file']);
+                    $traceLine['file'] = str_ireplace(SYSTEMPATH, 'SYSTEMPATH/', $traceLine['file']);
+                    if (defined('VENDORPATH')) {
+                        // VENDORPATH is not defined unless `vendor/autoload.php` exists
+                        $traceLine['file'] = str_ireplace(VENDORPATH, 'VENDORPATH/', $traceLine['file']);
+                    }
+                    $traceLine['file'] = str_ireplace(ROOTPATH, 'ROOTPATH/', $traceLine['file']);
```

Comment:
This could use `clean_path`
```suggestion
                    $traceLine['file'] = clean_path($traceLine['file']);
```

Reply:
@paulbalandan I have not made any other changes and this update is specifically to address the breaking changed that was released earlier. This can be addressed later separately.